spec/controllers/active_storage/direct_uploads_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ActiveStorage::DirectUploadsController do
```

Comment:
I decided to test the `ActiveStorage::BaseController` through `ActiveStorage::DirectUploadsController` as it's enough to exercise the error handling code and I believe it's the most urgent case.